### ISSUE TITLE
AllINONE+ IS as KM (forgot password and username) links not working

### ISSUE THREAD

**Issue by kalevivek11** [wso2/product-apim#8179]
Hi,
take wso2 3.0.0 and IS (5.9) as km do configurations
and try clicking these links forget password and username.
you will get errors below.
![image](https://user-images.githubusercontent.com/10954248/83144743-72289680-a111-11ea-8905-88e8997961f3.png)
![image](https://user-images.githubusercontent.com/10954248/83144802-840a3980-a111-11ea-911c-519f3f755fea.png)

**Comment by wasuradananjith** [comment]
Aa mentioned in [1] we will be removing **forgot username** from the UI and it will be added back once [1] is fixed. 

Forgot password was working in AllINONE pack and it worked without an issue. We will test this in 
AllINONE + IS as KM and will get back to you soon.

[1] https://github.com/wso2/product-apim/issues/8180

**Comment by kalevivek11** [comment]
@wasuradananjith can you confirm this is the bug?

**Comment by wasuradananjith** [comment]
@kalevivek11 I checked this. I did not get the error that you received. Did you use the **prepackaged WSO2 Identity Server as a Key Manager 5.9.0** as mentioned in the docs [here](https://apim.docs.wso2.com/en/3.0.0/install-and-setup/deploying-wso2-api-manager/distributed-deployment/configuring-wso2-identity-server-as-a-key-manager/)?

**Comment by kalevivek11** [comment]
yes @wasuradananjith did checked only km or ALLINONE APIM with IS as KM. which machine you are using windows or Linux or mac

**Comment by wasuradananjith** [comment]
I am using Linux (Ubuntu 18.04.4 LTS), will check again.

**Comment by kalevivek11** [comment]
@wasuradananjith try windows

**Comment by kalevivek11** [comment]
@wasuradananjith any update?

**Comment by kalevivek11** [comment]
@wasuradananjith did you try??
